src/components/TimeseriesChart/TimeseriesChart.tsx

```diff
@@ -94,6 +94,25 @@ export class TimeseriesChart extends Component<
     }
   };
 
+  getYdomainValues = (
+    ydomainConfig: any,
+    timeSeriesId: number = NaN
+  ): number[] | undefined => {
+    if (ydomainConfig === undefined) {
+      return undefined;
+    }
+    if (!isNaN(timeSeriesId)) {
+      const scaleOptions = ydomainConfig[timeSeriesId];
+      const min = scaleOptions ? scaleOptions.min : undefined;
+      const max = scaleOptions ? scaleOptions.max : undefined;
+      return scaleOptions && min && max ? [min, max] : undefined;
```

Comment:
What if `min` or `max` value is `0` here? And overall this method looks unreasonably complex.  I would suggest redesign it.